services/account_type/audit.yaml

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
```

Comment:
I know you're coming in cold @mattyellen. This template provisions the Role and Policy necessary to get access to the bucket that contains CloudTrail event data. An account that holds this data is called an "Audit" account. Per @silvexis, this does not need the Bucket Policy we currently provision.

See https://whimsical.com/GqcagtKp2N7CjZydBHriSn#2Ux7TurymMnhqMf7H736 for a bit of context.